runtime/lua/vim/lsp/util.lua

```diff
@@ -545,6 +545,12 @@ do
   local all_buffer_diagnostics = {}
 
   local diagnostic_ns = api.nvim_create_namespace("vim_lsp_diagnostics")
+  local sign_ns = 'vim_lsp_signs'
+
+  vim.fn.sign_define('LspDiagnosticsErrorSign',       {text=vim.g['LspDiagnosticsErrorSign'] or 'x', texthl='LspDiagnosticsError', linehl='', numhl=''})
```

Comment:
this could use better defaults than just "x", like the same you used in the help.